File: Exercicios/C/negativos.c
/*
Problema "negativos"
Faça um programa que leia um número inteiro positivo N (máximo = 10) e depois N números inteiros
e armazene-os em um vetor. Em seguida, mostrar na tela todos os números negativos lidos.
Exemplo:
Quantos numeros voce vai digitar? 6
Digite um numero: 8
Digite um numero: -2
Digite um numero: 9
Digite um numero: 10
Digite um numero: -3
Digite um numero: -7
NUMEROS NEGATIVOS:
-2
-3
-7
*/

#include <stdio.h>

int main(){

    int i, N;
    int vet [10];

    printf("Quantos numeros voce vai digitar? ");
    scanf("%d",&N);

    for (i = 0; i < N; i++) {
        printf("Digite um numero: ");
        scanf("%d",&vet[i]);
    }

    printf("NUMEROS NEGATIVOS:\n");

    for (i = 0; i < N; i++) {
        if (vet[i]<0) {
            printf("%d\n",vet[i]);
        }
    }
    return 0;
}

File: Exercicios/Java/Vetores/negativos.java
/*
Problema "negativos"
Faça um programa que leia um número inteiro positivo N (máximo = 10) e depois N números inteiros
e armazene-os em um vetor. Em seguida, mostrar na tela todos os números negativos lidos.
Exemplo:
Quantos numeros voce vai digitar? 6
Digite um numero: 8
Digite um numero: -2
Digite um numero: 9
Digite um numero: 10
Digite um numero: -3
Digite um numero: -7
NUMEROS NEGATIVOS:
-2
-3
-7
 */
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int N, i;

        System.out.print("Quantos numeros voce vai digitar? ");
        N = sc.nextInt();

        int[] vet = new int [N];

        for (i=0; i < N; i++) {
            System.out.print("Digite um numero: ");
            vet[i] = sc.nextInt();
        }
        sc.close();
        System.out.println("NUMEROS NEGATIVOS: ");
        for (i=0; i < N; i++) {
            if (vet[i] < 0) {
                System.out.println(vet[i]);
            }
        }

    }
}
